src/django_components_js/scripts/rollup.config.mjs

```diff
@@ -0,0 +1,59 @@
+import packageJson from '../package.json' assert { type: 'json' };
+
+import { babel } from '@rollup/plugin-babel';
+import terser from '@rollup/plugin-terser';
+import { nodeResolve } from '@rollup/plugin-node-resolve';
+import typescript from '@rollup/plugin-typescript';
+
+const extensions = ['.ts', '.tsx', '.js', '.jsx', '.es6', '.es', '.mjs'];
+const banner = `/*!
+* ${packageJson.name} v${packageJson.version}
+* Released under the MIT License.
+*/\n`;
+
+export default [
```

Comment:
Used [Rollup](https://rollupjs.org/) to build self-installing JS scripts that can be used like so:

```html
<script src="/path/to/script.js"></script>
```

If the JS package was published on NPM, it could be also loaded from a CDN like so:
```html
<script src="https://cdn.jsdelivr.net/npm/django-components@0.x.x/dist/cdn.min.js"></script>
```